copydb/config.go

```diff
@@ -44,6 +44,15 @@ type Config struct {
 	// Filter configuration for tables to copy
 	Tables FilterAndRewriteConfigs
 
+	// Hints in which order to create database tables, as <db>.<table> . If a
+	// table is to be created on start and appears in this list, it is created
+	// before any other table, and is created in the order listed here
+	//
+	// This is useful when tables have dependencies amongst each other. By
+	// default, create them in an arbitrary order. Names refer to original
+	// databases and tables - that is, before renaming occurs.
+	TableCreationOrder []string
```

Comment:
`TablesToBeCreatedFirstInOrder`?

Reply:
ACK. Will fix once I get more feedback to avoid unnecessary patch-sets (or once I get the "all clear otherwise" :-) )